sparse/etc/usb-moded/dyn-modes/99-developer_mode-usb.ini

```diff
@@ -0,0 +1,16 @@
+[mode]
```

Comment:
What about calling it developer_mode-native ?

Reply:
hmmm good idea, will do that as soon as the CI works again